python_modules/dagster/dagster/_daemon/asset_daemon.py

```diff
@@ -626,6 +653,43 @@ def _create_initial_sensor_cursors_from_raw_cursor(
 
         return result
 
+    def _rename_auto_materialize_states(
```

Comment:
this is more of a copy than a rename right? (which is good in the hopefully unlikely case that a user needs to roll back - the old state with the old name will still be there, harmless)

Reply:
that's correct yeah -- changed the comment here to be a bit more accurate and updated the name to _copy_default_auto_materialize_sensor_states.